typehero/typehero
# Evaluate code at time of submission

## Description

The changes in this PR re-evaluates `tsErrors` at time of submission. This is a workaround to #1695, where the solution is rejected on first log-in. Note that this solution will solve a similar class of problem, where users make changes to the code and quickly press submit.

## Looking Forward

As a next step, I think it would be nice to capture the code at the time of submission and only run the type-check on this code. The current changes does not account for the fact that users may change the code after pressing the submit button - however, I reason that this is quite unlikely.

## Related Issue




Closes #1695

## How Has This Been Tested?

1. Ensure that the original scenario in #1695 is not reproduceable
2. Add a random import statement and quickly press submit

cc @typesafeui
